rtdata/languages/default

```diff
@@ -3604,11 +3538,9 @@ TP_RAW_3PASSBEST;3-pass (Markesteijn)
 TP_RAW_4PASS;3-pass+fast
 TP_RAW_AHD;AHD
 TP_RAW_AMAZE;AMaZE
-TP_RAW_AMAZEBILINEAR;AMaZE+Bilinear
```

Comment:
`TP_RAW_AMAZEBILINEAR;AMaZE+Bilinear`
`TP_RAW_DCBBILINEAR;DCB+Bilinear`
`TP_RAW_RCDBILINEAR;RCD+Bilinear`
These three dual-demosaic keys are in use.

Reply:
Re-added in next commit.